### DISCUSSION TITLE
Title should be looking role instead of name

### DISCUSSION THREAD

**Discussion by o-be-one** [rendercv/rendercv#194] [Ideas]
Hello,

this is probably more a personal opinion. Imho best practice is to show the role we are looking for as title instead of our name. Like:
```
# DevOps
Name - Mail - Phone - Social networks
```

Why ? When HR get our CV, at this moment they don't care about our name. First thing they want to know if "what role we are looking for" and then they want to know our contact. Our name as no importance, its low priority... Until first real interviews (not the HR one, but technical one more, and closer to be hired).
That's why since years my CV show role first as a main title.

Maybe it depends as well to job we are applying, so maybe we could have a setting for that to set role as title or name as title.

Thank you :)

**Comment by sinaatalay** [comment]
Thank you. It's an interesting argument. Let's leave this issue open to see what other people think about it.

Also, please note that you can change the title to anything by overriding the `Header.j2.tex` and `Header.md` files. Please see [here](https://docs.rendercv.com/user_guide/#overriding-built-in-themes) for details.
